package.json

```diff
@@ -18,6 +19,7 @@
     "lint": "jshint js preprocessing validation",
     "jasmine-tests": "jasmine",
     "test": "npm run lint && npm run jasmine-tests && npm run validate",
-    "validate": "node validation/markets-validator.js"
+    "validate": "node validation/markets-validator.js",
+    "start": "http-server -f index.html"
```

Comment:
Does that mean the `http-server` is required to serve the production website because we removed the hash prefix from the path name part of the url?